tests/tests_unit/test_cdf_tk/test_loaders.py

```diff
@@ -1113,6 +1113,8 @@ def cognite_module_files_with_loader() -> Iterable[ParameterSet]:
         ).load_defaults(source_path)
         config = config_init.as_build_config()
         config.set_environment_variables()
+        # TODO: Consider other ways to identify modules
+        config.variables.pop("_cdf_tk", None)
```

Comment:
I think `# TODO: Consider other ways to identify modules` should instead be a task in Jira.

```suggestion
        # Todo Remove once the new modules in `_cdf_tk/prototypes/_packages` are finished.
        config.variables.pop("_cdf_tk", None)
```